endstate_correction/tests/test_simulation.py

```diff
@@ -67,7 +69,7 @@ def setup(tmp_path_factory, bss_protocol):
             env=env,
             ml_atoms=list(range(27)),
             protocol=bss_protocol,
-            name=system_name,
+            name='openMM',
```

Comment:
is there a reason you changed this variable assignment?

Reply:
In the BSSOrion, we use the name to identify the engine name, `lambda_0/amber.out` or `lambda_0/gromacs.log` so I'm doing this here as well to make sure that the same logic is being used.
That being said, this is the test, so it doesn't really affect the code base.

Reply:
thanks for clarifying!